vim-canvas-demo-app-react/functions/api/launch.ts

```diff
@@ -7,6 +7,11 @@ export const onRequestGet: PagesFunction<Env> = async (context) => {
   const queryParams = url.searchParams;
   const launchId = queryParams.get("launch_id");
   const launchType = queryParams.get("launch_type");
+  const ehrUrl = queryParams.get("ehr_url");
+  if (ehrUrl) {
```

Comment:
Add a more detailed comment about what the ehr_url is, and what it can be used for; Maybe even add a link to the relevant documentation